libp2p/protocols/pubsub/gossipsub.nim

```diff
@@ -170,7 +200,9 @@ method onNewPeer*(g: GossipSub, peer: PubSubPeer) =
   peer.iHaveBudget = IHavePeerBudget
   peer.pingBudget = PingsPeerBudget
 
-method onPubSubPeerEvent*(p: GossipSub, peer: PubSubPeer, event: PubSubPeerEvent) {.gcsafe.} =
+method onPubSubPeerEvent*(
```

Comment:
these are unrelated reformattings that should not be part of this PR

Reply:
@AlejandroCabeza could you please revert all formatting changes?

Reply:
Sure, I'll try to undo the stylings.

Reply:
I think the formatting when the code also changed is fine, I, at least, I was referring to the changes that are purely formatting related.